server/package.json

```diff
@@ -52,8 +49,6 @@
     "elevenlabs": "^0.5.0",
     "express": "^4.18.2",
     "extract-json-from-string": "^1.0.1",
-    "extract-zip": "^2.0.1",
-    "graphql": "^16.7.1",
```

Comment:
Same, no export of DB - why graphql is in here is anyones guess

Reply:
extract-zip from old export ability https://github.com/Mintplex-Labs/anything-llm/commit/6d651db6d3b585e398ce371d00a21190e816f799

graphql from Weaviate - need to retest
https://github.com/Mintplex-Labs/anything-llm/commit/f3a6147ffd5cd773fb6af97cc417b965b1678f01

Reply:
`graphql` is required by weaviate to work